src/workload.py

```diff
@@ -165,3 +166,13 @@ def get_service_pid(self) -> int:
                     return int(pid)
 
         raise snap.SnapError(f"Snap {self.SNAP_NAME} pid not found")
+
+    @override
+    def get_version(self) -> str:
+        if not self.kafka.present:
+            return ""
```

Comment:
**todo:** I think here, we shouldn't be checking if the Kafka 'snap' is present, but instead if it's actually running - aka 'can run a bin command and it works'.

I'd suggest doing `self.kafka.active`?